src/AutoPilotPlugins/PX4/PX4FlightBehaviorCopter.qml

```diff
@@ -0,0 +1,155 @@
+
+    Component {
+        id: pageComponent
+
+        Column {
+
+            spacing:            _margins
+
+            Column {
+                visible:                _sys_vehicle_resp
+
+                QGCCheckBox {
+                    id:                 responsivenessCheckbox
+                    text:               qsTr("Enable responsiveness slider (if enabled, acceleration limit parameters and others are automatically set)")
+                    checked:            _sys_vehicle_resp ? (_sys_vehicle_resp.value >= 0) : false
+                    onClicked: {
+                        if (checked) {
+                            _sys_vehicle_resp.value = Math.abs(_sys_vehicle_resp.value)
+                        } else {
+                            _sys_vehicle_resp.value = -Math.abs(_sys_vehicle_resp.value)
+                        }
+                    }
+                }
+
+                FactSliderPanel {
+                    width:          availableWidth
+                    enabled:        responsivenessCheckbox.checked
+
+                    sliderModel: ListModel {
+                        id:             responsivenessSlider
+
+                        ListElement {
+                            title:          qsTr("Responsiveness")
+                            description:    qsTr("A higher value makes the vehicle react faster. Be aware that this affects braking as well, and a combination of slow responsiveness with high maximum velocity will lead to long braking distances.")
+                            param:          "SYS_VEHICLE_RESP"
+                            min:            0.01
+                            max:            1
+                            step:           0.01
+                        }
+                    }
+                }
+                QGCLabel {
+                    visible:            _sys_vehicle_resp ? (_sys_vehicle_resp.value > 0.8) : false
+                    color:              qgcPal.warningText
+                    text:              "Warning: a high responsiveness requires a vehicle with large thrust-to-weight ratio. The vehicle might lose altitude otherwise."
+                }
+            }
+
+            Column {
+                visible:                _mpc_xy_vel_all
+
+                QGCCheckBox {
+                    id:                 xyVelCheckbox
+                    text:               qsTr("Enable horizontal velocity slider (if enabled, individual velocity limit parameters are automatically set)")
+                    checked:            _mpc_xy_vel_all ? (_mpc_xy_vel_all.value >= 0) : false
+                    onClicked: {
+                        if (checked) {
+                            _mpc_xy_vel_all.value = Math.abs(_mpc_xy_vel_all.value)
+                        } else {
+                            _mpc_xy_vel_all.value = -Math.abs(_mpc_xy_vel_all.value)
+                        }
```

Comment:
maybe:
`_mpc_xy_vel_all.value = Math.abs(_mpc_xy_vel_all.value) * (checked ? 1.0 : -1.0)`
or
```
const mult = checked ? 1.0 : -1.0;
_mpc_xy_vel_all.value = mult*Math.abs(_mpc_xy_vel_all.value)
```

Reply:
I was considering that but didn't really see a benefit.